### ISSUE TITLE
Windows Legendary(Epic Games) Error: UnicodeEncodeError

### ISSUE THREAD

**Issue by RichardBrowning** [Heroic-Games-Launcher/HeroicGamesLauncher#3033]
### Describe the bug

Some games do not launch, with an error like below:
```Console
* World War Z (App name: Wombat | Version: 2.21)[cli] INFO: Logging in...[Core] INFO: Trying to re-use existing login session...[cli] INFO: Getting game list... (this may take a while)Traceback (most recent call last):
File "legendary\cli.py", line 3069, in <module>File "legendary\cli.py", line 2980, in main
File "legendary\cli.py", line 239, in list_games
File "encodings\cp1252.py", line 19, in encode
UnicodeEncodeError: 'charmap' codec can't encode character '\u300a' in position 3: character maps to <undefined>
[14884] Failed to execute script 'cli' due to unhandled exception!
```

### Add logs

```text
(01:44:44) INFO:    [Legendary]:        Running command: XDG_CONFIG_HOME=C:\Users\mi888\AppData\Roaming\heroic\legendaryConfig C:\Users\mi888\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary --version
(01:44:44) INFO:    [Legendary]:        Legendary location: C:\Users\mi888\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary
(01:44:44) INFO:    [Gog]:              GOGDL location: C:\Users\mi888\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl
(01:44:44) INFO:    [Connection]:       Connectivity: check-online
(01:44:44) INFO:    [Connection]:       Pinging external endpoints
(01:44:44) INFO:    [Backend]:          DRM module staus {
  "neifaoindggfcjicffkgpmnlppeffabd": {
    "name": "Google Widevine Windows CDM",
    "status": "new",
    "version": "1.0.2512.1"
  },
  "oimompecagnajdejgnnjijobebaeigek": {
    "name": "Widevine Content Decryption Module",
    "status": "new",
    "version": "4.10.2557.0"
  }
}
(01:44:44) WARNING: [Backend]:          Protocol already registered.
(01:44:44) INFO:    [Connection]:       Connectivity: online
(01:44:44) ERROR:   [Gog]:              Unable to syncQueued playtime, userData not present
(01:44:44) INFO:    [Gog]:              Running command: C:\Users\mi888\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\gogdl --auth-config-path C:\Users\mi888\AppData\Roaming\heroic\gog_store\auth.json --version
(01:44:44) INFO:    [Backend]:          Loading Screen Ready
(01:44:44) INFO:    [Frontend]:         Refreshing undefined Library
(01:44:44) WARNING: [Backend]:          refresh not implemented on Sideload Library Manager
(01:44:44) INFO:    [Legendary]:        Refreshing library...
(01:44:44) INFO:    [Legendary]:        Refreshing Epic Games...
(01:44:44) INFO:    [Legendary]:        Game list updated, got 35 games & DLCs
(01:44:44) INFO:    [Backend]:          Frontend Ready
(01:44:45) WARNING: [Backend]:          listUpdateableGames not implemented on Sideload Library Manager
(01:44:45) INFO:    [Gog]:              Found 0 game(s) to update
(01:44:45) INFO:    [Nile]:             Running command: XDG_CONFIG_HOME=C:\Users\mi888\AppData\Roaming\heroic\nile_config C:\Users\mi888\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\nile --version
(01:44:45) INFO:    [Backend]:          Checking for new Heroic Updates
(01:44:45) INFO:    [Backend]:          Checking for current version changelog
(01:44:45) INFO:    [Legendary]:        Running command: XDG_CONFIG_HOME=C:\Users\mi888\AppData\Roaming\heroic\legendaryConfig C:\Users\mi888\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary list --third-party
(01:44:45) INFO:    [Legendary]:        Checking for game updates: XDG_CONFIG_HOME=C:\Users\mi888\AppData\Roaming\heroic\legendaryConfig C:\Users\mi888\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary list --third-party
(01:44:46) INFO:    [Legendary]:        Found 0 games to update
(01:44:46) DEBUG:   [Backend]:          Unable to load Steam Libraries, libraryfolders.vdf not found
(01:44:49) WARNING: [Backend]:          Could not determine System Info
(01:44:50) INFO:    [Backend]:          Starting the Download Queue
(01:45:21) INFO:    [Legendary]:        Getting more details with 'legendary info'
(01:45:22) INFO:    [Legendary]:        Running command: XDG_CONFIG_HOME=C:\Users\mi888\AppData\Roaming\heroic\legendaryConfig C:\Users\mi888\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary info 19927295d6e3467887d4e830d8c85963 --platform Windows --json
(01:45:23) DEBUG:   [Backend]:          {"free":428999331840,"diskSize":510745636864,"message":"399.54 GiB / 475.67 GiB","validPath":true}
(01:45:25) DEBUG:   [Legendary]:        Using cached install info
(01:45:25) INFO:    [DownloadManager]:  Absolute Drift  was added to the download queue.
(01:45:25) INFO:    [Backend]:          Preventing machine to sleep
(01:45:25) DEBUG:   [Legendary]:        Using cached install info
(01:45:25) INFO:    [Legendary]:        Installing 19927295d6e3467887d4e830d8c85963: XDG_CONFIG_HOME=C:\Users\mi888\AppData\Roaming\heroic\legendaryConfig C:\Users\mi888\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary install 19927295d6e3467887d4e830d8c85963 --platform Windows --base-path C:\Users\mi888\Games\Heroic --skip-dlcs --skip-sdl -y
(01:45:25) DEBUG:   [Legendary]:        Logging to file "C:\Users\mi888\AppData\Roaming\heroic\GamesConfig\19927295d6e3467887d4e830d8c85963.log"
(01:45:25) DEBUG:   [Backend]:          {"free":428998799360,"diskSize":510745636864,"message":"399.54 GiB / 475.67 GiB","validPath":true}
(01:45:27) INFO:    [Legendary]:        Progress for Absolute Drift: 13.22%/19.92MB/00:00:04 Down: 19.69MB/s / Disk: 77.96MB/s
(01:45:28) INFO:    [Legendary]:        Progress for Absolute Drift: 29.46%/44.39MB/00:00:01 Down: 24.1MB/s / Disk: 105.38MB/s
(01:45:29) INFO:    [Legendary]:        Progress for Absolute Drift: 47.33%/71.32MB/00:00:01 Down: 26.82MB/s / Disk: 28.99MB/s
(01:45:30) INFO:    [Legendary]:        Progress for Absolute Drift: 84.78%/127.77MB/00:00:01 Down: 55.78MB/s / Disk: 62.1MB/s
(01:45:31) INFO:    [Legendary]:        Progress for Absolute Drift: 100%/150.70MB/00:00:00 Down: 22.89MB/s / Disk: 78.91MB/s
(01:45:32) INFO:    [Legendary]:        installed.json updated, refreshing library
(01:45:32) INFO:    [Legendary]:        installed.json updated, refreshing library
(01:45:32) INFO:    [Backend]:          Adding shortcuts for Absolute Drift
(01:45:32) INFO:    [DownloadManager]:  Finished Installation of 19927295d6e3467887d4e830d8c85963
(01:45:32) INFO:    [DownloadManager]:  19927295d6e3467887d4e830d8c85963 added to download manager finished.
(01:45:32) INFO:    [DownloadManager]:  19927295d6e3467887d4e830d8c85963 removed from download manager.
(01:45:32) INFO:    [Frontend]:         Refreshing legendary Library
(01:45:32) INFO:    [Legendary]:        Refreshing library...
(01:45:32) INFO:    [Legendary]:        Refreshing Epic Games...
(01:45:32) INFO:    [Legendary]:        Game list updated, got 35 games & DLCs
(01:45:32) INFO:    [Backend]:          operation interrupted
(01:45:33) INFO:    [Legendary]:        Running command: XDG_CONFIG_HOME=C:\Users\mi888\AppData\Roaming\heroic\legendaryConfig C:\Users\mi888\AppData\Local\Programs\heroic\resources\app.asar.unpacked\build\bin\win32\legendary list --third-party
```

### Steps to reproduce

1. Log in to Epic Game account
2. Click into Library
3. Install any game
4. Run the game installed
5. See error

### Expected behavior

Runs The game correctly.

### Screenshots

![Screenshot 2023-09-09 014913](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/46521212/f3bc1c43-60b1-4045-affc-c4701610f8e4)

### Heroic Version

Latest Stable

### System Information

- OS : Windows 11 Build 22621.2215
-Heroic Version : 2.9.1 stable 

### Additional information

It is likely this is an encoding problem that might only happen to people who install specific games containing the "illegal char" in the title.

**Comment by flavioislima** [comment]
@RichardBrowning check if the fix for this issue here works for you:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/issues/526

**Comment by Birdie0** [comment]
Can legendary version shipped with heroic be upgraded to `0.20.33`? it still comes with `0.20.32` which causes this bug.

**Comment by arielj** [comment]
this should be fixed in heroic 2.10.0 since we upgraded legendary
